libs/cms/src/lib/cms.resolver.ts

```diff
@@ -609,3 +612,20 @@ export class FeaturedSupportQNAsResolver {
     )
   }
 }
+
+@Resolver(() => PowerBiSlice)
+@Directive(cacheControlDirective())
+export class PowerBiSliceResolver {
+  constructor(private powerBiService: PowerBiService) {}
+
+  @ResolveField(() => graphqlTypeJson)
```

Comment:
Field name can be simpler like "embedProps" and have a proper GraphQL type instead of using graphqlTypeJson. I.e. two fields: accessToken and embedUrl.

Reply:
Agreed